Week-9/todo-app/src/App.js

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { Card } from 'react-bootstrap';
+import FormTodo from './components/FormTodo';
+import Todo from './components/Todo';
+import './App.css';
+
+function App() {
+  const [todos, setTodos] = useState([
+    {
+      text: 'This is a sample todo 1',
+      isDone: false,
+    },
+    {
+      text: 'This is a sample todo 2',
+      isDone: true,
```

Comment:
why are having default todo's?

Reply:
Just to play around with the app. Should I remove this?